<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="25.94s"
		durationms="25940"
		type="root"
		rid="12e11d28ea681fc"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="I-Ray"
		rxversion="10.5.1.0"
		host="IRAYSERVER"
		osversion="Windows 10 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		oslanguage="en-US"
		dtnlanguage="en-US"
		screenresolution="1600x900"
		timestamp="10/18/2023 6:41:51 PM"
		timestampiso="2023-10-18T13:11:51"
		endtime="10/18/2023 6:42:17 PM"
		timeoutfactor="1"
		runid=""
		sutversion=""
		module="Fittingvalues">
		<detail>
			Fittingvalues
		</detail>
		<varbindings>
		</varbindings>
		<item
			timeRelativeToTestSuiteStartTime="00:25.936"
			timeRelativeToTestModuleStartTime="00:25.936"
			timeWallClock="18:42:17"
			time="00:25.936"
			level="Error"
			category="Module">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
